tests/pytorch/nightly/resnet50-mp.libsonnet

```diff
@@ -93,6 +94,67 @@ local utils = import 'templates/utils.libsonnet';
       },
     },
   },
+  local resnet50_tpu_vm = experimental.PyTorchTpuVmMixin {
+    frameworkPrefix: 'pt-nightly',
+    modelName: 'resnet50-mp',
+    paramsOverride: {
+      num_epochs: error 'Must set `num_epochs`',
+      datadir: error 'Must set `datadir`',
+    },
+    command: utils.scriptCommand(
+      |||
+        %(setup_commands)s
+        pip3 install tensorboardX google-cloud-storage
+        python3 xla/test/test_train_mp_imagenet.py \
+          --logdir=$(MODEL_DIR) \
+          --datadir=%(datadir)s \
+          --model=resnet50 \
+          --num_workers=8 \
+          --batch_size=128 \
+          --log_steps=200 \
+          --num_epochs=%(num_epochs)d \
+      ||| % self.paramsOverride,
+    ),
+    podTemplate+:: {
+      spec+: {
+        containerMap+: {
+          train+: {
+            resources+: {
+              requests: {
+                cpu: '1',
+                memory: '2Gi',
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+  local functional_tpu_vm = common.Functional {
+    paramsOverride: {
+      setup_commands: common.tpu_vm_nightly_install,
```

Comment:
Should setup_commands also be in resnet50_tpu_vm?

Reply:
I added them to the `paramsOverride` object such that they'll throw an error if not provided. However the setup commands were there in the yamls when I ran the manual test